client/wc-api/wp-data-store/create-api-client.js

```diff
@@ -19,7 +19,9 @@ function createStore( name ) {
 function createDataHandlers( store ) {
 	return {
 		dataRequested: resourceNames => {
-			if ( document.hidden ) {
+			const { resources } = store.getState();
+			const newResources = resourceNames.filter( resourceName => ! resources[ resourceName ] );
```

Comment:
We can use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here, a little bit more performant and it returns a bool:
```suggestion
			const newResources = resourceNames.some( resourceName => ! resources[ resourceName ] );
```

Reply:
Sweet- thanks!